ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/SingleEntityVulnerabilitiesTable.tsx

```diff
@@ -1,44 +1,69 @@
 import React from 'react';
 import { Button, ButtonVariant } from '@patternfly/react-core';
-import { TableComposable, Thead, Tr, Th, Tbody, Td } from '@patternfly/react-table';
+import {
+    TableComposable,
+    Thead,
+    Tr,
+    Th,
+    Tbody,
+    Td,
+    ExpandableRowContent,
+} from '@patternfly/react-table';
 import { CheckCircleIcon, ExclamationCircleIcon } from '@patternfly/react-icons';
 import { SVGIconProps } from '@patternfly/react-icons/dist/js/createIcon';
 
 import LinkShim from 'Components/PatternFly/LinkShim';
 import SeverityIcons from 'Components/PatternFly/SeverityIcons';
+import useSet from 'hooks/useSet';
 import { vulnerabilitySeverityLabels } from 'messages/common';
 import { getDistanceStrictAsPhrase } from 'utils/dateUtils';
 import { ImageVulnerabilitiesResponse } from '../hooks/useImageVulnerabilities';
 import { getEntityPagePath } from '../searchUtils';
+import ImageComponentsTable from './ImageComponentsTable';
+import { ImageDetailsResponse } from '../hooks/useImageDetails';
 
 export type SingleEntityVulnerabilitiesTableProps = {
+    image: ImageDetailsResponse['image'] | undefined;
     imageVulnerabilities: ImageVulnerabilitiesResponse['image']['imageVulnerabilities'];
 };
 
 function SingleEntityVulnerabilitiesTable({
+    image,
     imageVulnerabilities,
 }: SingleEntityVulnerabilitiesTableProps) {
+    const expandedRowSet = useSet<string>();
     return (
         <TableComposable>
             <Thead>
                 <Tr>
+                    <Th>{/* Header for expanded column */}</Th>
                     <Th>CVE</Th>
                     <Th>Severity</Th>
-                    <Th>CVE Status</Th>
+                    <Th>CVE status</Th>
```

Comment:
Bravo!

Reply:
All credit to @alwayshooin for pointing this one out.